ui/src/app/applications/components/application-deployment-history/application-deployment-history.tsx

```diff
@@ -20,6 +20,7 @@ export const ApplicationDeploymentHistory = ({
     rollbackApp: (info: models.RevisionHistory) => any;
     selectDeployment: (index: number) => any;
 }) => {
+    console.log('selectedRollbackDeploymentIndex: ', selectedRollbackDeploymentIndex);
```

Comment:
Is log added intentionally here or maybe a miss?

Reply:
Good catch! Fixed.